phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/apps/ViewApps.java

```diff
@@ -79,13 +81,28 @@ public void showAvailableApps() {
      */
     public void populate(List<CategoryDTO> categories) {
         Platform.runLater(() -> {
+            Set<String> themedCategories = Sets.newHashSet(
+                    "custom",
```

Comment:
Yes can. But are not necessarily (e.g. from some custom repository). I need to figure out for which ones the CSS can be used. I tried several things but could not find a better solution.